lib/resource/Group.js

```diff
@@ -6,37 +6,29 @@ var utils = require('../utils');
 function Group() {
   Group.super_.apply(this, arguments);
 }
+
 utils.inherits(Group, DirectoryChildResource);
 
 Group.prototype.addAccount = function addGroupAccount(/* accountOrAccountHref, [options,] callback */) {
-  var args = Array.prototype.slice.call(arguments);
-  var account = args.shift();
-  var callback = args.pop();
-  var options = (args.length > 0) ? args.shift() : null;
+  var args = utils.resolveArgs(arguments, ['account', 'options', 'callback']);
```

Comment:
This scenario doesn't seem to work. The `options` argument should be optional but it isn't, so calling `.addAccount()` without `options` will result in an error.